utils/__tests__/conversions.test.ts

```diff
@@ -339,13 +339,6 @@ describe("ApiBonusActivationFromModel", () => {
 });
 
 describe("ApiBonusVacanzaBaseFromModel", () => {
-  it("should not decode an invalid domain object", () => {
-    const invalidDomainObject = { dsuRequest: { familyMembers: [] } };
-    // @ts-ignore needed to test an unrepresentable type assignment
-    const result = toApiBonusVacanzaBase(invalidDomainObject);
```

Comment:
this test looks useless as-is (in which scenario do we pass an invalid domain object to this method ?)